types.go

```diff
@@ -0,0 +1,79 @@
+package fq
+
+import (
+	"math"
+)
+
+// TODO(aaron-prindle) currently testing with one concurrent request
+const C = 1 // const C = 300
+
+// TODO(aaron-prindle) currently service time "G" is not implemented entirely
+const G = 100000 //   100000 nanoseconds = 100 milliseconds || const G = 60000000 nanoseconds = 1 minute
+
+type Packet struct {
+	// request   http.Request
+	item interface{}
+	// virfinish float64
+	size int
+	// key       uint64
+	queueidx  int
+	seq       int
+	starttime float64
```

Comment:
Better make this `startTime time.Time`, it is another place where we are trying to measure the passage of real time.

Reply:
Updated